docs/data/sql_functions.yml

```diff
@@ -1098,6 +1099,15 @@ json:
 
       -- {"K": null}
       JSON_OBJECT('K' VALUE JSON(''))
+      
+      -- [{"K2":42}]
+      JSON_ARRAY(JSON('{"K2": 42}'))
+
+      -- [{"K2":{"K3":42}}]
+      JSON_ARRAY(JSON('{"K2":{"K3":42}}'))
```

Comment:
I think it would be good to give an example with more than one element

Reply:
Hey David, I've addressed your concern 👍